modules/recaptcha/modules.php

```diff
@@ -54,9 +54,8 @@ protected function output() {
  */
 class Hm_Output_recaptcha_form extends Hm_Output_Module {
     protected function output() {
-        if (!$this->get('router_login_state')) {
-            $rconf = $this->get('recaptcha_config');
-            return '<div class="g-recaptcha" data-sitekey="'.$this->html_safe($rconf['site_key']).'"></div>';
+        if (!$this->get('router_login_state') && $this->get('recaptcha_config')) {
+            return '<div class="g-recaptcha" data-sitekey="'.$this->html_safe($this->get('recaptcha_config')['site_key']).'"></div>';
```

Comment:
this error comes back when I remove the condition that I put
[Sun Jul 21 11:18:55.635757 2024] [php:warn] [pid 1948:tid 1952] [client ::1:7190] PHP Warning:  Trying to access array offset on value of type null in C:\\xampp\\htdocs\\my_cypht\\modules\\recaptcha\\modules.php on line 59